### ISSUE TITLE
Clang format?

### ISSUE THREAD

**Issue by fire** [WonderMediaProductions/Maya2glTF#81]
I recommend using clang format and picking a style so I don't have to worry about making whitespace changes.

**Comment by Ziriax** [comment]
Sounds good, I did that initially but got some weird formatting here and there, so I disabled it. But you are right, a style is needed

**Comment by fire** [comment]
Can you pick a style, it doesn't matter the particulars, but you are the maintainer.

**Comment by Ziriax** [comment]
Clang with default settings is fine.

**Comment by fire** [comment]
Put a `.clang-format` file in the git root

and

```
# We'll use defaults from the LLVM style, but with 4 columns indentation.
BasedOnStyle: LLVM
IndentWidth: 4
```

I'll make a pr with `clang-format -i *` in src/

**Comment by Ziriax** [comment]
Fixed by #84
